src/Helpers/LayoutHelper.php

```diff
@@ -32,6 +32,16 @@ public static function isLayoutTopnavEnabled()
         return config('adminlte.layout_topnav') || View::getSection('layout_topnav');
     }
 
+    /**
+     * Check if preloader is enabled.
+     *
+     * @return bool
+     */
+    public static function isPreloaderEnabled()
+    {
+        return config('adminlte.preloader');
```

Comment:
May you add a default value of `false`, just in case the configuration do not exists, like:

`config('adminlte.preloader', false)`